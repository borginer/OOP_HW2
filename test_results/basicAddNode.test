CreateGraph G
CreateNode A 5
AddNode G A
ListNodes G
