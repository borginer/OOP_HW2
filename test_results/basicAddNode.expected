created graph G
created node A with cost 5
added node A to G
G contains: A
